src/appmixer/helpscout/core/FindConversations/component.json

```
@@ -36,10 +37,46 @@
                         "index": 1,
                         "tooltip": "Advanced Helpscout customer query. Example: `(subject:\"have a question\" AND tag:\"billing\")`. See the <a target=_blank href=\"https://developer.helpscout.com/mailbox-api/endpoints/conversations/list/#query\">docs</a> for the full query syntax."
                     },
+                    "status": {
+                        "type": "select",
+                        "label": "Status",
+                        "index": 2,
+                        "tooltip": "Filter conversations by status. Defaults to active.",
+                        "options": [
+                            {
+                                "clearItem": true,
+                                "content": "-- Clear selection --"
+                            },
+                            {
+                                "label": "Active",
+                                "value": "active"
+                            },
+                            {
```

By default, the API returns only active conversations, so leaving this unselected doesn't return all (which I would expect when I don't use a filter)
I think it would be better if you made the select like this:
1. Remove option to clear selection
2. Add "defaultValue": "all", or "active"

```suggestion
                    "status": {
                        "type": "select",
                        "label": "Status",
                        "index": 2,
                        "tooltip": "Filter conversations by status. Defaults to active.",
                        "defaultValue": "active",
                        "options": [
                            {
                                "label": "Active",
                                "value": "active"
                            },
                            {
```

---

Since the api already defaults to active we do not have to send the value "active" and the tooltip says "defaults to active" should suffice